scripts/filters/default-injects.js

```diff
@@ -21,20 +21,5 @@ hexo.extend.filter.register('theme_inject', (injects) => {
     if (filePath[key]) {
       injects[key].push(filePath[key]);
     }
-    // Compatible, but like head header etc, it shouldn't add 's' suffix.
-    let oldKey = key + 's';
-    if (filePath[oldKey]) {
-      injects[key].push(filePath[oldKey]);
```

Comment:
I think we shouldn‘t delete the compatibility code here now.

Reply:
It's a draft PR and will be merged at least one month later (to let the warning message display in version v7.3.0 and v7.4.0)

Reply:
How about create a new milestone like v8.0.0